<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Renamer</class>
 <widget class="QMainWindow" name="Renamer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="browse_button">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="directory_path_edit">
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1777215</width>
        <height>35</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="confirm_button">
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="fetch_button">
      <property name="text">
       <string>Fetch</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QTreeWidget" name="changelist">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Old Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Name</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
